opentelemetry-api/exporter/TrackMetric/src/main/java/TrackMetric.java

```diff
@@ -1,22 +1,34 @@
 import com.azure.monitor.opentelemetry.exporter.AzureMonitorExporterBuilder;
 import com.azure.monitor.opentelemetry.exporter.implementation.SemanticAttributes;
-import io.opentelemetry.api.OpenTelemetry;
 import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.DoubleGauge;
 import io.opentelemetry.api.metrics.DoubleHistogram;
 import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
 import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk;
 import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdkBuilder;
 
 import java.util.UUID;
 
 public class TrackMetric {
     private static final String CONNECTION_STRING = "<Your Connection String>";
-    private static final OpenTelemetry openTelemetry = initOpenTelemetry();
+    private static final OpenTelemetrySdk openTelemetry = initOpenTelemetry();
     private static final Meter meter = openTelemetry.getMeter("my meter");
 
     public static void main(String[] args) throws InterruptedException {
-        trackDoubleHistogram();
+//        trackDoubleHistogram();
+//        Thread.sleep(5000); // wait at least 5 seconds to give batch span processor time to export
+
+        trackGauge();
+    }
+
+    public static void trackGauge() throws InterruptedException {
+        // track a gauge
+        DoubleGauge gauge = meter.gaugeBuilder("gauge" + UUID.randomUUID()).build();
+        gauge.set(111.0);
         Thread.sleep(5000); // wait at least 5 seconds to give batch span processor time to export
+
+        openTelemetry.shutdown();
```

Comment:
probably clearer to move openTelemetry.shutdown() to the end of the `main` method